modules/plottinggl/include/modules/plottinggl/rendering/boxselectionrenderer.h

```diff
@@ -65,13 +65,17 @@ class IVW_MODULE_PLOTTINGGL_API BoxSelectionRenderer {
     const BoxSelectionProperty& settings_;
     LineSettings lineSettings_;
     algorithm::LineRenderer lineRenderer_;
-    vec4 color_;  //!< Dummy color until line color can be set using uniform
-    ColoredMesh dragRectMesh_ = ColoredMesh(DrawType::Lines, ConnectivityType::Strip,
-                                            {{vec3{0.f, 0.f, 0.f}, vec4{0.5f, 0.5f, 0.5f, 1.f}},
-                                             {vec3{1.f, 0.f, 0.f}, vec4{0.5f, 0.5f, 0.5f, 1.f}},
-                                             {vec3{1.f, 1.f, 0.f}, vec4{0.5f, 0.5f, 0.5f, 1.f}},
-                                             {vec3{0.f, 1.f, 0.f}, vec4{0.5f, 0.5f, 0.5f, 1.f}}},
-                                            {0, 1, 2, 3, 0});
+
+    using PositionMesh = TypedMesh<buffertraits::PositionsBuffer2D>;
+    PositionMesh dragRectMesh_{DrawType::Lines,
```

Comment:
might be static?

Reply:
There is a call to set the model matrix in the draw call...
 `dragRectMesh_.setModelMatrix(m);`